demo.README.md

```diff
@@ -0,0 +1,98 @@
+---
+language: 
+- ru
+- en
+thumbnail: https://raw.githubusercontent.com/JetRunner/BERT-of-Theseus/master/bert-of-theseus.png
+tags:
+- translation
+- fsmt
+license: Apache 2.0
```

Comment:
see https://huggingface.co/docs#what-metadata-can-i-add-to-my-model-card

> License identifiers are those standardized by GitHub here.

Reply:
Right, see I'm still wrong. 

"License identifiers are those standardized by GitHub here." is very vague - do we want the keyword "apache-2.0" or the full string "Apache license 2.0"?

Have a look at the current cards:

```
$ grep -hr license: model_cards | sort | uniq
license: ""
license:  Apache-2.0
license: Apache 2.0
license: Apache-2.0
license: "apache-2.0"
license: apache-2.0
license: cc-by-nc-4.0
license: MIT
license: "MIT"
license: mit
license: "mit"
license: mit
```

Reply:
yaml removes non-necessary quotes, and I guess we could auto-lowercase, so there's no that much variation there

Again, mid-term goal is to have built-in validation!

Reply:
Oh yes and by identifiers I meant the keywords in the keyword column: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/licensing-a-repository#searching-github-by-license-type

Reply:
If I may share my experience - The problem is not so much whether one could automate the normalization of tags. The problem is that it's very confusing to the writer of the card of what is expected, especially when there is such a wide variety of tags that do the same. Should I copy from this file, or that one. Hence my suggestion would be to fix all the cards to do it in an identical way - i.e. above there should be only 3 licenses - and not 9. 

Thank you.